documentation/manual/src/doc/algebras/json-schemas.md

```diff
@@ -105,6 +105,18 @@ It will work similarly for other representations of enumerated values.
 Most of them provide `values` which can conveniently be passed into `enumeration`.
 However, it is still possible to explicitly pass a certain subset of allowed values.
 
+### Tuples
+
+JSON schemas for tuples from 2 to 22 elements are provided out of the box. For instance, if
+there are implicit `JsonSchema` instances for types `A`, `B`, and `C`, then you can summon
+a `JsonSchema[(A, B, C)]`. Tuples are modeled in JSON with arrays, as recommended in the
+[JSON Schema documentation](https://json-schema.org/understanding-json-schema/reference/array.html#tuple-validation).
+
+Here is an example of JSON schema for a GeoJSON `Point`, where GPS coordinates are modeled with a pair (longitude, latitude):
+
+~~~ scala src=../../../../../json-schema/json-schema/src/test/scala/endpoints/algebra/JsonSchemasDocs.scala#tuple
+~~~
+
```

Comment:
This looks like this:

![Screenshot from 2019-10-31 16-24-33](https://user-images.githubusercontent.com/332812/67960693-f3ab5b00-fbfa-11e9-9a45-cd95eb59b2db.png)